docs/configuration-validation.md

```diff
@@ -17,3 +17,17 @@ If you were presented with configuration error (or a warning, depending on `conf
 **Note**: In a warning mode (with `configValidationMode: warn` set in configuration) Framework commands are not blocked in any way, e.g. `sls deploy` will still attempt to deploy the service normally (still depending on the source of the warning, success of a deployment may vary)
 
 If you find this functionality problematic, you may also turn it off with `configValidationMode: off` setting.
+
+## Configuration
+
+### Where
+
+At the root of the `serverless.yml` file, add a `configValidationMode` key.
+
+### Supported Values
+
+| Value | Effect                                                                                        |
+|-------|-----------------------------------------------------------------------------------------------|
+| error | Print configuration errors and fail packaging. As of serverless v3 this is the default value. |
```

Comment:
This is not accurate - the default is `warn` not `error`

Let's write this as `Fail invoked command and output configuration errors` - it doesn't have to be about packaging

Reply:
Ah, my bad. I reread the release notes to confirm.

> Configuration validation is still kept at the "warning" level by default (instead of turning to errors, as initially planned). 

👍